src/main/java/com/squareup/kotlinpoet/ClassName.kt

```diff
@@ -137,12 +138,10 @@ class ClassName private constructor(
       return ClassName(names)
     }
 
-    @JvmStatic fun get(clazz: KClass<*>): ClassName {
-      val qualifiedName = clazz.qualifiedName
-      if (qualifiedName == null) {
-        throw IllegalArgumentException("$clazz cannot be represented as a TypeName")
-      }
-      return ClassName.bestGuess(qualifiedName)
+    @JvmStatic @JvmName("get")
```

Comment:
Do you care about the JVM name here? For code in another JVM language to have a KClass sounds unlikely